docs/release/2022q1.md

```diff
@@ -0,0 +1,144 @@
+## 2022-04-01: 2022q1 release
+
+This release covers everything from 2022-01-01 and has been tested with avr-gcc
+v11.2.0 from Upstream and arm-none-eabi-gcc 10.3-2021.10 from Arm.
+
+Breaking changes:
+
+None.
+
+Features:
+
+- STM32L5 Support.
+- Raspberry Pi PR2040 Support.
+- Atmel SAM Timer driver.
+- Improved Project and Doxygen documentation.
+- STM32G4 internal flash driver.
+
+Integrated Projects:
+
+- ETL added at v20.27.1.
+- TinyUSB upgraded to v0.13.0.
+- LVGL upgraded to v8.2.0.
+
+Fixes:
+
+- Issues when using TinyUSB and FreeRTOS together solved.
+- Fix SPI bug in handling of CPHA/CPOL on Atmel SAM.
+- Read unique ID correctly on SAMG55.  
+- Fix GPIO peripheral connect method on Atmel SAM.
```

Comment:
```suggestion
- Fix GPIO peripheral connect method on Atmel SAM.
- Heap allocator defaults to Newlib on devices with one continuous RAM section.
- FreeRTOS now always always supports static allocation.
```

(tiny, but important change in fa44651)

Reply:
There's no `Atmel` anymore, I'll update it to `Microchip`.